01_Intro/Guess the Number/Guess the Number/Program.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Guess_the_Number
+{
+    class Program
+    {
+        enum difficulty { Easy = 100, Normal = 500, Hardcore = 1000 }
+        static void Main(string[] args)
+        {
+            Random rnd = new Random();
+            int ret = GetDifficulty();
+            int cheat = -322;
+            int dif = 0;
+            string diff = "";
+            switch (ret)
+            {
+                case 1:
+                    diff = "Easy";
+                    dif = Convert.ToInt16(difficulty.Easy);
+                    break;
+                case 2:
+                    diff = "Normal";
+                    dif = Convert.ToInt16(difficulty.Normal);
+                    break;
+                case 3:
+                    diff = "Hardcore";
+                    dif = Convert.ToInt16(difficulty.Hardcore);
+                    break;
+                default:
+                    Console.WriteLine("Wrong input.");
+                    break;
+            }
+            int rand = rnd.Next(1, dif);
```

Comment:
hier kommt dann einmal die Zeile ```Convert.ToInt16(dif ) ``` aber als parameter

Reply:
die variable die du hier dann neu brauchst ist dann ```ret```. Das Ergebnis davon würde ich nochmal kurz in eine Variable packen. Das sorgt dafür das du dies eine Zeile weiter unten nochmal verwenden kannst.